src/ParseServer.js

```diff
@@ -196,7 +196,7 @@ class ParseServer {
     api.use('/health', function (req, res) {
       res.status(options.state === 'ok' ? 200 : 503);
       if (options.state === 'starting') {
-        res.set('Retry-After', 1);
+        res.set('Retry-After', 100);
```

Comment:
Should we use a central variable to manage the retry-after policy? I think we have recently added this response header somewhere else too, right?

Reply:
What do you think we should name it?

Reply:
> This also increases the retry-after header from 1ms to 100ms, as normally the server takes around 3s-4s to start, and 1ms results in the endpoint being called too often

@dblythy shouldn't this delay be in seconds instead of ms per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After? If so, 100 seems too high

Reply:
Nice pickup!!